cmd/commands/subvolume.go

```diff
@@ -0,0 +1,41 @@
+package command
+
+import (
+	"github.com/rook/kubectl-rook-ceph/pkg/subvolume"
+	"github.com/spf13/cobra"
+)
+
+// SubvolumeCmd represents the rook command to manage subvolumes
+var SubvolumeCmd = &cobra.Command{
+	Use:   "stale-subvolume",
+	Short: "manage subvolumes",
```

Comment:
```suggestion
	Short: "manage subvolumes",
```
Please describe it in more words as this is what the users see in the terminal for command description.